### ISSUE TITLE
There are many dependencies in the idea software that need to be re-joined

### ISSUE THREAD

**Issue by mingtianshi** [goxr3plus/XR3Player#81]

**Comment by goxr3plus** [comment]
What do you mean my friend . I have given all the instructions on the read me :) . Idea software is just what i use , you can do it in Eclipse too .

**Comment by mingtianshi** [comment]
~~~
		<!-- CHROMIUM BROWSER -->
		<dependency>
			<groupId>com.teamdev.jxbrowser</groupId>
			<artifactId>jxbrowser-win64</artifactId>
			<version>6.23.1</version>
		</dependency>
 ~~~
I'm sorry, but this place doesn't seem to have this dependency pack, but there's a version 7.12.2

The maven version I use is 3.6.3

**Comment by goxr3plus** [comment]
Well they took down that dependency from Maven central? That's very strange. Well the version 7.12.2 is paid. Mmmmm it looks that I have too remove completely the browser from the App. In the meantime I have another branch With JXBrowser 7  can you have a look on that if it works for you :)? I will have a look on XR3Player tommorow to see about the JxBrowser :)

**Comment by HelgeStenstrom** [comment]
I'm not a user of XR3Player, but I'm interested in how it's build, as use it as inspiration. I haven't opened the project in my IDE (IntelliJ) for a long time. Now it seems to be broken, and the problem is related to jxbrowser. The first problem is 
https://github.com/goxr3plus/XR3Player/blob/3bdda1a9b4c2e80eb648b4fc055c41587b96d4e6/src/main/java/module-info.java#L47
which is marked in red (Module not found: jxbrowser). I kind understand a part of it. Like you say above, there  is no version 6.23.1 in Maven Central, only 7.12.2. See https://mvnrepository.com/artifact/com.teamdev.jxbrowser/jxbrowser.
But I have version 6.23.1 in my local ~/.m2/repository, so there should be no need for Maven to go out on the net to look for this dependency. I'm running on Mac; I don't know if that matters.

If I made XR3Player for my own needs, I would remove the browser. But it's not my project.

**Comment by goxr3plus** [comment]
I agree with you guyz I need to remove it from XR3Player it needs some work it was working perfectly before they release version 7 and it broke everything because of the license.


I will put effort on it to remove it completely soon.

**Comment by goxr3plus** [comment]
I am just too busy recently with my Playstore apps and work. Give me some time.

**Comment by goxr3plus** [comment]
Hello everyone today i cloned again XR3Player locally and i am working on updating all the dependencies and removing JXBrowser . I will notify here and commit the changes  when it's ready .
